axelor-admin/src/main/java/com/axelor/apps/base/service/app/DataBackupRestoreService.java

```diff
@@ -131,4 +138,46 @@ public boolean SeuencesExist() {
     long total1 = (long) JPA.em().createQuery("SELECT count(*) FROM MrpLineType").getSingleResult();
     return total > 0 || total1 > 0 ? true : false;
   }
+
+  public Object importObjectWithByteArray(Object bean, Map<String, Object> values)
+      throws IOException {
+    assert bean instanceof Model;
+    final Path path = (Path) values.get("__path__");
+
+    for (String fieldName : getByteArrFields(values)) {
```

Comment:
No need of getByteArrFields method as its not used anywhere else.  Same loop can be used here to avoid double looping.